vms/platformvm/vm_test.go

```diff
@@ -736,70 +736,47 @@ func TestRewardValidatorAccept(t *testing.T) {
 	abort := options[1].(*blockexecutor.Block)
 	require.IsType(&block.BanffAbortBlock{}, abort.Block)
 
-	require.NoError(oracleBlk.Accept(context.Background()))
+	// Assert block tries to reward a genesis validator
+	rewardTx := oracleBlk.(block.Block).Txs()[0].Unsigned
+	require.IsType(&txs.RewardValidatorTx{}, rewardTx)
```

Comment:
let's introduce the assertion that would have make us spot the error in the first place